### ISSUE TITLE
The USER must be specified

### ISSUE THREAD

**Issue by h4kc** [hadolint/hadolint#1025]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
When the USER is not specified in a docker file, it should trigger a warning because the user will be root. 
### Actual behavior
The DL3002 warning is checking if the last user is not a root. But if USER is not set the user of the container is root and warning is triggred 
### Steps to reproduce the behavior

Any docker file not having USER set.

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 2.12.1
```

**Comment by AlexFabre** [comment]
Indeed, DL3002 seems to be raised only if a `USER root` instruction appears in the dockerfile.
Without any USER instruction DL3002 is never raised. 

DL3002 should rise on both conditions:
- `USER root` is the last USER instruction 
- No `USER` instruction at all.
